modules/030-cloud-provider-azure/docs/CONFIGURATION.md

```diff
@@ -22,6 +22,8 @@ StorageClass Configuration Parameters:
 
 * `provision` — sets additional StorageClasses for [Azure ultra disks](https://docs.microsoft.com/en-us/azure/virtual-machines/disks-types#ultra-disk):
   * `name` — the name of the class to create;
+  * `type` — SKU type;
```

Comment:
Why not `skuName` to make it similar to original csi driver?

Reply:
Done.

Reply:
Sorry, I think it was a bad decision to do these changes. We have type field in all other providers, so it's better to keep `type`. I suggest to change description to `azure disk storage account type`, add link to docs https://docs.microsoft.com/en-us/azure/aks/azure-disk-csi#storage-class-driver-dynamic-disk-parameters and provide available types with additional information for users https://docs.microsoft.com/en-us/azure/storage/common/storage-account-overview#types-of-storage-accounts.

Reply:
Done.